src/api/ws.ts

```diff
@@ -1,29 +1,19 @@
-import { IpcMain, WebContents } from 'electron';
+import { WebContents } from 'electron';
 import WebSocket from 'ws';
 
-import { JsonRPCResponse } from '@types';
+import { handleRequest } from './api';
 
-import { handleRequest, handleResponse } from './api';
-
-export const runAPI = (ipcMain: IpcMain, webContents: WebContents) => {
+export const runAPI = (webContents: WebContents) => {
   console.debug('Spinning up WS');
```

Comment:
Creating and use `Logger` | `LogService` for all you debug calls.
Or use an Express/Node log library